NOnion/Constants.fs

```diff
@@ -114,13 +114,12 @@ module Constants =
     let internal RotationTimeOffset = TimeSpan.FromHours 12.0
 
     let internal Ed25519BasePointString =
-        "(15112221349535400772501151409588531511454012693041857206046113283949847762202,46316835694926478169428394003475163141307993866256225615783033603165251855960)"
+        "(15112221349535400772501151409588531511454012693041857206046113283949847762202, 46316835694926478169428394003475163141307993866256225615783033603165251855960)"
         |> Encoding.ASCII.GetBytes
 
     let internal HiddenServiceBlindString =
-        "Derive temporary signing key"
+        "Derive temporary signing key" + string Char.MinValue
```

Comment:
not a fan of this way to interpolate strings, let's use sprintf? btw wrt commit message, what does "More" exactly mean? do you mean "More info"?

Reply:
BTW if putting links to a git repo, better to put of a commit hash one, otherwise the line numbers might get outdated

Reply:
Links updated, and added the missing word "info"